basics/a73.go

```diff
@@ -0,0 +1,18 @@
+// Loops and if conditions
+// basic if condition
+package main
+
+import "fmt"
+
+func main() {
+	var num1, num2 int
+	num1 = 52
+	num2 = 53
+	if (num1 < num2) {
+		fmt.Printf("%d is less than %d", num1, num2)
```

Comment:
The output is: 52 is less than 53%

Notice the trailing '%'. Add a new line '/n' to get rid of it since fmt.Printf does not add a new line.